client/styles.css

```diff
@@ -13,6 +13,26 @@
 /*
 
 =====================
+PROFILE
+=====================
+
+*/
+.profile-height {
+  height: 600px;
+}
```

Comment:
Yeah, on that note. Noticed when we reduce the screen size, the gradient section of the page cuts off the content and we're left with a big chunk of white down below. Just something to add to backlog. I know we said we wouldn't worry about mobile responsiveness but it should be responsive for web I think. 👍